src/collectd.conf.pod

```diff
@@ -5487,6 +5487,7 @@ Synopsis:
       Host "localhost"
       Socket "/var/run/mysql/mysqld.sock"
       SlaveStats true
+      MariaDb false
```

Comment:
Can we please name the option more descriptive? Like `MultiSourceReplication`?

Reply:
the variable includes mariadb dialect, multisource-replication on the vanilla version is determined automatically

Reply:
maybe MariaDbMultiSourceReplication?, or let's leave it as it is for the future.

Reply:
I am very in favor of having a more speaking name. MariaDb sounds very generic and may lead to misunderstandings.

Reply:
maybe MariadbDialect, like telegraf (https://github.com/influxdata/telegraf/blob/master/plugins/inputs/mysql/mysql.go) ?